app/views/docs/database.phtml

```diff
@@ -453,5 +453,183 @@ func main() async throws {
     </tbody>
 </table>
 
+<h3><a href="/docs/database#pagination" id="ordering">Ordering Results</a></h3>
+<p>When querying using the <a href="/docs/server/database#databaseListDocuments">listDocuments</a> endpoint, you can specify the order of the documents returned by providing the <b>orderAttributes</b> and <b>orderTypes</b> parameters.</p>
```

Comment:
Let's also document what is default behaviour - if you don't provide any sorting data.. I think it's by ID ASC (creating date). And to reverse it, you need to do `$id` `DESC`. All just my thinking, let's double-check.